rewrite-weblogic/documentation/content/Jakarta Server Faces 3x.md

```diff
@@ -0,0 +1,101 @@
+
+### Tags:
+- jakarta
+- jakartaee
+- faces
+- jsf
+
+### Recipe Source
+
+[jakarta-faces-3.yaml](https://github.com/oracle/rewrite-recipes/blob/main/rewrite-weblogic/src/main/resources/META-INF/rewrite/jakarta-faces-3.yaml)
+
+### Recipe List:
+
+- [org.openrewrite.java.migrate.jakarta.JakartaFacesEcmaScript](https://docs.openrewrite.org/recipes/java/migrate/jakarta/jakartafacesecmascript)
+- [org.openrewrite.java.migrate.jakarta.RemovedUIComponentConstant](https://docs.openrewrite.org/recipes/java/migrate/jakarta/removeduicomponentconstant)
+- com.oracle.weblogic.rewrite.jakarta.JakartaFaces3WebXml
+- com.oracle.weblogic.rewrite.jakarta.JakartaFaces3Xhtml
+- com.oracle.weblogic.rewrite.jakarta.JavaxFacesConfigXmlToJakartaFaces3ConfigXml
+- com.oracle.weblogic.rewrite.jakarta.JavaxFacesTagLibraryXmlToJakartaFaces3TagLibraryXml
+- com.oracle.weblogic.rewrite.jakarta.JavaxWebFragmentXmlToJakartaWebFragmentXml5
+- com.oracle.weblogic.rewrite.jakarta.JavaxWebXmlToJakartaWebXml5
+- com.oracle.weblogic.rewrite.jakarta.FacesJNDINamesChanged3
+- com.oracle.weblogic.rewrite.jakarta.RemovedJakartaFaces3ExpressionLanguageClasses
+- com.oracle.weblogic.rewrite.jakarta.RemovedJakartaFaces3ResourceResolver
+- com.oracle.weblogic.rewrite.jakarta.RemovedStateManagerMethods3
+- com.oracle.weblogic.rewrite.jakarta.FacesManagedBeansRemoved3
+- com.oracle.weblogic.rewrite.jakarta.UpgradeFacesOpenSourceLibraries3
+
+### Usage
+
+This recipe will find and replace legacy JSF namespaces and `javax` references with Jakarta Server Faces values.
+
+#### Maven POM
+
+1. Add the following to your Maven POM file.
+```
+<plugin>
+    <groupId>org.openrewrite.maven</groupId>
+    <artifactId>rewrite-maven-plugin</artifactId>
+    <version>5.43.0</version>
+    <configuration>
+        <exportDatatables>true</exportDatatables>
+        <activeRecipes>
+            <recipe>com.oracle.weblogic.rewrite.Faces2xMigrationToJakarta3x</recipe>
+        </activeRecipes>
+    </configuration>
+    <dependencies>
+        <dependency>
+            <groupId>com.oracle.weblogic.rewrite</groupId>
+            <artifactId>rewrite-weblogic</artifactId>
+            <version>0.4.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.openrewrite.recipe</groupId>
+            <artifactId>rewrite-migrate-java</artifactId>
+            <version>2.28.0</version>
+        </dependency>
+    </dependencies>
+</plugin>
+...
+```
+2. Run `mvn rewrite:run` to run the recipe.
+
+#### Gradle
+
+1. Add the following to your `build.gradle` file:
+
+```
+plugins {
+    id("org.openrewrite.rewrite") version("7.1.4")
+}
+
+rewrite {
+    activeRecipe("com.oracle.weblogic.rewrite.Faces2xMigrationToJakarta3x")
+    setExportDatatables(true)
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    rewrite("org.openrewrite.recipe:rewrite-migrate-java:3.2.0")

```

Comment:
Is missing to add a line for `rewrite("com.oracle.weblogic.rewrite:rewrite-weblogic:0.4.2")`